### DISCUSSION TITLE
Is there a way for me to make this pop up on stream only when a song change happens.

### DISCUSSION THREAD

**Discussion by RetroRey** [dlrudie/Snip#520] [Q&A]
Im attempting to set this up on my stream, ive got it working fine. however i dont want to display at all times but rather on song change i would like it to pop in for a small time and then disappear.

has anyone figured out how to do this or know if this is possible.

Thanks in advance.

**Comment by GenesisFR** [comment]
That's beyond the scope of Snip. You should look into streaming plugins.
